javascript/index.js

```diff
@@ -1,65 +1,109 @@
-let splits = document.getElementById('splits');
-
-function printTime() {
-  // ... your code goes here
-}
-
-function printMinutes() {
-  // ... your code goes here
-}
-
-function printSeconds() {
-  // ... your code goes here
-}
-
-// ==> BONUS
-function printMilliseconds() {
-  // ... your code goes here
-}
-
-function printSplit() {
-  // ... your code goes here
-}
-
-function clearSplits() {
-  // ... your code goes here
-}
-
-function setStopBtn() {
-  // ... your code goes here
-}
-
-function setSplitBtn() {
-  // ... your code goes here
-}
-
-function setStartBtn() {
-  // ... your code goes here
-}
-
-function setResetBtn() {
-  // ... your code goes here
-}
-
-// Start/Stop Button
-btnLeft.addEventListener('click', () => {
-  // ... your code goes here
-});
+window.addEventListener("load", () => {
+  const chronometer = new Chronometer();
+
+  // get the buttons:
+  const btnLeft = document.getElementById('btnLeft');
+  const btnRight = document.getElementById('btnRight');
+
+  // get the DOM elements that will serve us to display the time:
+  let minDec = document.getElementById('minDec');
+  let minUni = document.getElementById('minUni');
+  let secDec = document.getElementById('secDec');
+  let secUni = document.getElementById('secUni');
+  let milDec = document.getElementById('milDec');
+  let milUni = document.getElementById('milUni');
+  let splits = document.getElementById('splits');
+
+  function printTime() {
+    printMinutes();
+    printSeconds();
+  }
+
+  function printMinutes() {
+    let min = chronometer.twoDigitsNumber(chronometer.getMinutes());
+    minDec.innerText = min[0];
+    minUni.innerText = min[1];
+  }
+
+  function printSeconds() {
+    let sec = chronometer.twoDigitsNumber(chronometer.getSeconds());
+    secDec.innerText = sec[0];
+    secUni.innerText = sec[1];
+  }
+
+  // ==> BONUS
+  function printMilliseconds() {
+    let ms = chronometer.twoDigitsNumber(chronometer.msTime);
+    milDec.innerText = ms[0];
+    milUni.innerText = ms[1];
+  }
+
+  function printSplit() {
+    let split = document.createElement("li");
+    split.innerText = chronometer.splitClick();
+    splits.appendChild(split);
+  }
+
+  function clearSplits() {
+    splits.innerHTML = "";
+  }
 
-// Reset/Split Button
-btnRight.addEventListener('click', () => {
-  // ... your code goes here
+  function setStopBtn() {
+    btnLeft.classList.remove("start");
+    btnLeft.classList.add("stop");
+    btnLeft.innerText = "STOP";
+  }
+
+  function setSplitBtn() {
+    btnRight.classList.remove("reset");
+    btnRight.classList.add("split");
+    btnRight.innerText = "SPLIT";
+  }
+
+  function setStartBtn() {
+    btnLeft.classList.remove("stop");
+    btnLeft.classList.add("start");
+    btnLeft.innerText = "START";
+  }
+
+  function setResetBtn() {
+    btnRight.classList.remove("split");
+    btnRight.classList.add("reset");
+    btnRight.innerText = "RESET";
+  }
+
+  let isActive = false;
```

Comment:
Very clever going around this isActive boolean, you are already thinking in react ;) 
A simpler approach would be to check the current element class for the conditional, however I like your approach better, it is more future proof haha.